cilium/cmd/policy_wait.go

```diff
@@ -38,7 +38,7 @@ var policyWaitCmd = &cobra.Command{
 			Fatalf("invalid revision '%s': %s", args[0], err)
 		}
 
-		haveWaited := false
+		timeWaited := 0
```

Comment:
Did you consider expressing this instead as something like:

```
startTime := time.Now()
failDeadline := time.Now().Add(failWaitTime * time.Second)
maxDeadline := time.Now().Add(maxWaitTime * time.Second)
...
} else if time.Now().After(failDeadline) && notReady > 0 {
...
} else if time.Now().After(maxDeadline) {
```

Then, the log messages can also do something like "failed after %s ..", time.Since(startTime).

Reply:
Thanks for the ready-to-use suggestion :-)
Done.